controller/src/main/java/org/jboss/as/controller/capability/registry/RequirementRegistration.java

```diff
@@ -29,6 +37,7 @@
  */
 public class RequirementRegistration {
 
+    protected final Map<PathAddress, RegistrationPoint> registrationPoints = new LinkedHashMap<>();
```

Comment:
yeah it should be private. It needed to be protected before refactoring, that is no longer the case.